tests/02_eradiate/01_unit/scenes/atmosphere/test_particle_layer.py

```diff
@@ -168,9 +168,9 @@ def test_particle_layer_eval_ckd_scattering_only(
     layer = ParticleLayer(dataset=scattering_only)
     spectral_config = CKDMeasureSpectralConfig(bin_set="10nm", bins=bins)
     spectral_ctx = spectral_config.spectral_ctxs()[0]
-    assert np.all(layer.eval_sigma_s(spectral_ctx).magnitude > 0.0)
-    assert np.all(layer.eval_sigma_a(spectral_ctx).magnitude == 0.0)
-    assert np.all(layer.eval_albedo(spectral_ctx).magnitude == 1.0)
+    assert np.allclose(layer.eval_sigma_s(spectral_ctx), 0.2 / ureg.km)
+    assert np.allclose(layer.eval_sigma_a(spectral_ctx), 0.0 / ureg.km)
+    assert np.allclose(layer.eval_albedo(spectral_ctx).m, 1.0)
```

Comment:
The use of `all()` here was intentional: we wanted those values to be *exactly* equal to 0 or 1, not close.